client/package.json

```diff
@@ -15,8 +15,10 @@
     "extends": "react-app"
   },
   "dependencies": {
+    "axios": "^0.19.0",
```

Comment:
I used fetch API instead in the registration component

Reply:
Yeah, I noticed :+1:  
I can make a commit to switch to axios if you want me to do so.